src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -116,6 +115,11 @@ public void TestValidDiskSpaceProperties()
                 if (fbUser != drive.AvailableFreeSpace)
                     Assert.True(drive.AvailableFreeSpace >= 0);
 
+                // valid property getters shouldn't throw
+                string name = drive.Name;
+                string format = drive.DriveFormat;
```

Comment:
We could make a manual interop call to statfs/GetVolumeInformation and then compare the result to what is returned from drive.DriveFormat, but that would just be comparing the same call to itself. 

Beyond that I don't know of anything consistent that we could test for since we don't have any solid information on the drive we're testing other than the fact that it is valid.